core/client/app/components/gh-posts-list-item.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend(ActiveLinkWrapper, {
     }),
 
     authorAvatar: computed('post.author.image', function () {
-        return this.get('post.author.image') || this.get('ghostPaths.url').asset('/shared/img/user-image.png');
+        return this.get('post.author.image') || `${this.get('ghostPaths.subdir')}/ghost/img/user-image.png`;
```

Comment:
Nope, I don't know any. I suppose they shouldn't rely on it anyway since it could change any time. Cool, nice find / fix.